### ISSUE TITLE
isCacheableValue doesn't seem to work

### ISSUE THREAD

**Issue by lukechilds** [node-cache-manager/node-cache-manager#71]
```js
const cacheManager = require('cache-manager');
const memoryCache = cacheManager.caching({
  store: 'memory',
  isCacheableValue: val => val !== 'bar'
});

memoryCache.set('foo', 'bar')
  .then(memoryCache.get('foo'))
  .then(console.log);
  // "bar"
```

If I set `isCacheableValue` to return false for `'bar'` I can still cache it. Am I using it incorrectly?

**Comment by BryanDonovan** [comment]
Ah, that is a bit confusing.  The `isCacheableValue` param is only used in the `wrap` method.  I suppose it could be used in the `set` method too.  Hmm...

**Comment by lukechilds** [comment]
Thanks for clearing that up 👍 

For my use case I think I'll probably manually do the checks, so it makes no difference to me if you choose to enable `isCacheableValue ` on `set` or not. However I think if you don't support checking `isCacheableValue ` on `set` you should probably mention that in the readme as it's not obvious.

**Comment by botika** [comment]
Fixes by #203. `IsCacheable` is responsibility of Store, i.e., all set methods has `isCacheable` implemented in the respective store
